src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,609 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=1380182218&format=interactive",
+  sheets_gid="2032682744",
+  sql_file="correlation_cwv_preload.sql"
+  )
+}}
+
+This same observation may be seen on a page's <a hreflang="en" href="https://web.dev/lcp">Largest Contentful Paint</a>, indicating that in many cases, the developer is prioritizing resources which aren't needed to render the LCP element and as a consequence degrading the user experience.
+
+While this doesn't prove that having preload hints causes a page to get slower, having many does coincide with having slower performance. Every page has its unique requirements and it is impossible to apply a "one size fits all" approach, but in the majority of cases the number of preloaded resources should be kept low and resource prioritization should be delegated to the browser when possible.
+
+<p class="note">Note: In addition to the number of hints, the size of each preloaded resource has an impact on the website performance. The above figure does not take into consideration the size of each preloaded resource.</p>
+
+## `rel="preload"`
+
+With that being said, and the expectation that more websites will adopt `preload`, let's take a better look at the preload resource hint and understand why it is so effective, yet at the same time so prone to misuse.
+
+### The `as` attribute
+
+The `as` attribute should be specified when using `rel="preload"` (or `rel="prefetch"`) to specify the type of resource being downloaded. Applying the correct `as` attribute allows the browser to prioritize the resource more accurately. For example, `preload as="script"` will get a low or medium priority, while `preload as="style"` would be assigned an internal request priority of _Highest_. The `as` attribute is required for caching the resource for future requests and applying the correct <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP">Content Security Policy</a>.
+
+{{ figure_markup(
+  image="preload-as-attribute-values.png",
+  caption='`rel="preload" as` attribute values.',
+  description='Bar chart showing the usage of values for rel="preload". script is used 54.6% of the time on mobile and 54.5% on desktop. font is used 21.9% of the time on mobile and 23.5% on desktop. style is used 10.9% of the time on mobile and 11.3% on desktop. fetch is used 9.4% of the time on mobile and 7.5% on desktop. image is used 2.8% of the time on mobile and 2.8% on desktop.',
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTgxCYoH2vDbhTky1qQKEeV034kReHF8JYYq0aDyoo6LG22XL6Tar3dNPsqU1_zyvR6vuFEdMGoF1oP/pubchart?oid=1844544440&format=interactive",
```

Comment:
Remove "2021" from "Resource Hints 2021" in the subtitle and update the screenshot